src/main/webapp/gast/stat.jsp

```diff
@@ -0,0 +1,170 @@
+    String filterTitle = "";
+    if (request.getParameter("filterTitle") != null) {
+       filterTitle = request.getParameter("filterTitle");
+    }
+    /*Set Sort Type*/
+    String sort = "";
+    if (request.getParameter("sort") != null) {
+       sort = request.getParameter("sort");
+    }
+    /*Set jumpToID for Quelle*/
+    String jumpToID = "";
+    if (request.getParameter("jumpToID") != null) {
+       jumpToID = request.getParameter("jumpToID");
+    }
+    /*
+    Calculation for pagination
+     */
+    int currentPage = 1;
+
+    if (request.getParameter("currentPage") != null) {
+        currentPage = Integer.parseInt(request.getParameter("currentPage"));
+    }
+
+    int recordsPerPage = Constants.RECORDS_PER_PAGE;
+
+    if (request.getParameter("recordsPerPage") != null) {
+        recordsPerPage = Integer.parseInt(request.getParameter("recordsPerPage"));
+    }
+
+    int rows = QuelleDB.countStat(filterTitle).intValue();
+
+    //row count is 1 if users performs id search
+    if(jumpToID != null && jumpToID.length() > 0){
+        rows = 1;
+    }
+    
+    int nOfPages = rows / recordsPerPage;
+
+    if (nOfPages % recordsPerPage > 0) {
+        nOfPages++;
+    }
+    
+    List<Quelle> lst = QuelleDB.getList(currentPage, recordsPerPage,filterTitle,sort, jumpToID);
+%>
+<%!
+        
+    /*
+    Functions that helps to print pagination
+     */
+    public String html_prev_button(String filterTitle, String recordsPerPage, Integer currentPage, String sort, String jumpToID) {
+        return "<li class=\"page-item\"><a class=\"page-link\" href=\"?jumpToID="+jumpToID+"&sort="+sort+"&page=stat&filterTitle=" + filterTitle + "&recordsPerPage=" + recordsPerPage + "&currentPage=" + (currentPage - 1) + "\">Previous</a></li>";
+    }
+
+    public String html_next_button(String filterTitle, String recordsPerPage, Integer currentPage, String sort, String jumpToID) {
+        return "<li class=\"page-item\"><a class=\"page-link\"href=\"?jumpToID="+jumpToID+"&sort="+sort+"&page=stat&filterTitle=" + filterTitle + "&recordsPerPage=" + recordsPerPage + "&currentPage=" + (currentPage + 1) + "\">Next</a></li>";
+    }
+
+    public String html_page_item_current(int page) {
+        return "<li class=\"page-item active\"><a class=\"page-link\">" + page + "</a></li>";
+    }
+
+    public String html_page_item(int page, String filterTitle, int recordsPerPage, String sort, String jumpToID) {
+        return "<li class=\"page-item\"><a class=\"page-link\"href=\"?jumpToID="+jumpToID+"&sort="+sort+"&page=stat&filterTitle=" + filterTitle + "&recordsPerPage=" + recordsPerPage + "&currentPage=" + page + "\">" + page + "</a></li>";
+    }
+
+    public String html_sort_title_up(int page, String filterTitle, int recordsPerPage, String jumpToID) {
+        return "<a class=\"sort-link\"href=\"?jumpToID="+jumpToID+"&sort=titleUp&page=stat&filterTitle=" + filterTitle + "&recordsPerPage=" + recordsPerPage + "&currentPage=" + page + "\">sort up</a>";
```

Comment:
"sort up" + "sort down" should also be translated

Reply:
yes, fixed!